src/stylesheets/components/_card.scss

```diff
@@ -25,22 +32,20 @@
   .sds-card__header-actions {
     display: flex;
   }
-
 }
 
 .sds-card .sds-card__content {
   padding: 15px 0 0;
   display: block;
-
   @media only screen and (max-width: 600px) {
     display: none;
   }
 }
-
 //search fieldset
+
 .sds-search {
- display: block;
- margin: auto;
+  display: block;
+  margin: auto;
 
   .usa-button {
     background-color: #3F57A6;
```

Comment:
Prefer to use the token value and mixin/utils to grab color, background color, margin, padding, etc.
(see https://designsystem.digital.gov/utilities/ for all available)

`  .usa-button {
    @include u-bg('indigo-cool-60');
  }`

`  .usa-button {
    background-color: color('indigo-cool-60');
  }`